src/fool_ai_detector/service/roberta_base_openai_detector.py

```diff
@@ -0,0 +1,26 @@
+"""
+Evaluator based on the roberta_base_openai_detector
```

Comment:
```suggestion
Evaluator based on the roberta_base_openai_evaluator
```
We call it evaluator.

Reply:
Only model name, will resolve this.